### PR TITLE
Update plan-external-collaboration.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement that protected documents can be opened by external if permissions are configured.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8725

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/plan-external-collaboration.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Admins can apply a [retention policy](/microsoft-365/compliance/retention) on a
 
 ### Sensitivity labels
 
-[Sensitivity labels](/microsoft-365/compliance/sensitivity-labels) available in the host organization are the only labels that can be applied to the documents in a shared channel site. A file that is encrypted by a sensitivity label cannot be opened by external participants. Automatic labeling is not used.
+[Sensitivity labels](/microsoft-365/compliance/sensitivity-labels) available in the host organization are the only labels that can be applied to the documents in a shared channel site. A file that is encrypted by a sensitivity label cannot be opened by external participants unless permissoins are granted. Automatic labeling is not used.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/plan-external-collaboration.md]:
```suggestion
[Sensitivity labels](/microsoft-365/compliance/sensitivity-labels) available in the host organization are the only labels that can be applied to the documents in a shared channel site. A file that is encrypted by a sensitivity label cannot be opened by external participants unless permissions are granted. Automatic labeling is not used.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**